lms/static/coffee/src/instructor_dashboard/util.coffee

```diff
@@ -1,3 +1,4 @@
+# TODO - lots of safe-commit-linter failures in this file - how can I get HtmlUtils into a coffeescript file?
```

Comment:
If it's not using requirejs, you can use `edx.HtmlUtils`.  If that doesn't work, it may be a question for @andy-armstrong.